internal/controlplane/xds_routes.go

```diff
@@ -45,6 +45,8 @@ func (srv *Server) buildPomeriumHTTPRoutes(options *config.Options, domain strin
 		srv.buildControlPlanePathRoute("/healthz"),
 		srv.buildControlPlanePathRoute("/.pomerium"),
 		srv.buildControlPlanePrefixRoute("/.pomerium/"),
+		srv.buildControlPlanePathRoute("/.well-known"),
+		srv.buildControlPlanePrefixRoute("/.well-known/"),
```

Comment:
should we limit this to `/.well-known/pomerium/`?

Reply:
Yes, good idea.